ramls/_schemas/kv_configuration.schema

```diff
@@ -30,6 +30,15 @@
     "userId": {
       "type": "string"
     },
+    "recordsSource": {
+      "type": "string",
+      "default": "Source record storage",
+      "enum": [
+        "Source record storage",
+        "Inventory",
+        "Source record storage and Inventory"
+      ]
+    },
```

Comment:
New properties should be accompanied by a minor interface version change (both in the RAML and module descriptor).